build.gradle

```diff
@@ -151,7 +132,7 @@ task release(type: Copy, group: 'build') {
     from(zipTree(project.tasks.bundlePlugin.outputs.files.getSingleFile()))
     into "build/plugins/opendistro-knn"
```

Comment:
Why not change this to opensearch? Is that outside scope of PR?

Reply:
Thats correct. I tried to rename only obvious quoted strings that will not have impact on code.